cicecore/cicedynB/dynamics/ice_dyn_eap.F90

```diff
@@ -558,6 +558,10 @@ subroutine eap (dt)
                            field_loc_NEcorner, field_type_vector)
       call grid_average_X2Y('U2TF',strocnxT)    ! shift
       call grid_average_X2Y('U2TF',strocnyT)
+      if (grid_system == 'CD') then
```

Comment:
Please add a comment before this block (everywhere that it occurs), e.g.
> ! shift velocity components from CD grid locations (N, E) to B grid location (U) for transport

Reply:
As just discussed, it's probably better to comment out the code changes external to EVP itself, for now.